packages/foreman/rubygem-rubyipmi/rubygem-rubyipmi.spec

```diff
@@ -19,6 +19,7 @@ Requires: ipmitool
 Requires: %{?scl_prefix_ruby}ruby(release)
 Requires: %{?scl_prefix_ruby}ruby
 Requires: %{?scl_prefix_ruby}ruby(rubygems)
+Requires: %{?scl_prefix}rubygem(highline)
```

Comment:
This was added but it looks like this is only in Rakefile. I opened https://github.com/logicminds/rubyipmi/issues/54 for this.

Reply:
Is there any concern in this regard? I mean, we have that dependency already so whats the deal?

Reply:
Not a huge amount, but fewer dependencies is generally speaking easier. Remember that the Smart Proxy pulls in this gem as a dependency and then in turn another. That means a container image would also become larger and even on my Smart Proxy only RPM installation I don't have it installed. Luckily highline itself has no (runtime) dependencies.